src/app_level/font.c

```diff
@@ -0,0 +1,1250 @@
+#include "common.h"
+
+#define __ALIGNER_DF static u8 __CONCAT2__(aligner, __LINE__)[] = { 0xDF, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
snap uses `f3dex2` right?
If that's the case then this line is a `Gfx` with a single `gsSPEndDisplayList()`.
It is known that some tool from the sdk emitted empty display lists to force alignment of textures.
Using an actual `Gfx` is better than this because it force the next symbol to be 0x8 aligned.